data/io.github.zyedidia.micro.metainfo.xml

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component>
-	<id>com.github.zyedidia.micro</id>
+	<id>io.github.zyedidia.micro</id>
 	<launchable type="desktop-id">micro.desktop</launchable>
 	<name>Micro Text Editor</name>
 	<summary>A modern and intuitive terminal-based text editor</summary>
 	<metadata_license>MIT</metadata_license>
+	<project_license>MIT</project_license>
 	<categories>
 		<category>Development</category>
 		<category>TextEditor</category>
 	</categories>
 	<provides>
 		<binary>micro</binary>
+		<id>com.github.zyedidia.micro</id>
```

Comment:
Should this be `io.github.zyedidia.micro` if the other one is too?

Reply:
In commit message:
> We can use `<provides><id>...</id></provides>` for everyone who expected 
the old application_id

This isn't actually required since nothing was relying on the application id, but just in case it is probably worth having here